src/app.ts

```diff
@@ -1461,6 +1461,14 @@ export const initializeAppModules = () => {
       },
     });
 
+    addModule({

```

Comment:
The test app doesn't have a CI/CD pipeline so you need to build the changes locally and check-in the resulting html and js files.